sim/src/main/makefrag/midasexamples/Makefrag

```diff
@@ -30,23 +30,38 @@ else
 	firesim_sbt_project := {file:${firesim_base_dir}/}firesim
 endif
 
+# FAT_JAR is applicable across config tuples but is tied to the sbt project
+FAT_JAR := $(firesim_base_dir)/generated-src/firesim.jar
+PRE_ELABORATION_TARGETS += $(FAT_JAR)
+
 chisel_src_dirs = \
 		$(addprefix $(firesim_base_dir)/,. midas midas/targetutils firesim-lib) \
 		$(addprefix $(chipyard_dir)/generators/, chipyard rocket-chip/src, rocket-chip/api-config-chipsalliance)
 
 chisel_srcs = $(foreach submodule,$(chisel_src_dirs),\
 	$(shell find $(submodule)/ -iname "[!.]*.scala" -print 2> /dev/null | grep 'src/main/scala'))
 
+# don't search for implicit rules to make the chisel_srcs (reduces useless output in --debug=i)
+$(chisel_srcs): ;
+
+$(FAT_JAR):  $(chisel_srcs) $(SCALA_BUILDTOOL_DEPS)
```

Comment:
Why do we care about shipping FAT jars for the fastedtests and midasexamples? Or is this just done to keep things consistent across projects.

Reply:
I was just trying to keep things consistent.  Even though it isn't needed right now, I didn't think it was worth having them be different.